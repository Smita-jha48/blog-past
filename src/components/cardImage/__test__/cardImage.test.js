import { render } from '@testing-library/react';
import CardImage from '../cardImage';

describe('CardImage', () => {
  it('should render correctly', () => {
    const img = { image: 'filepath' };
    const { asFragment } = render(<CardImage {...img} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
