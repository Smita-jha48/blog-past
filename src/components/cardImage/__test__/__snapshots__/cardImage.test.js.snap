// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImage should render correctly 1`] = `
<DocumentFragment>
  <div
    class="img"
  >
    <img
      alt="img"
      src="filepath"
    />
  </div>
</DocumentFragment>
`;
